src/amber/support/file_encryptor.cr

```diff
@@ -0,0 +1,37 @@
+require "./message_encryptor"
+
+module Amber::Support
+  module FileEncryptor
+    def self.global_encryption_key
+      ENV["AMBER_ENCRYPTION_KEY"]? || begin
+        if File.exists?(".encryption_key")
+          File.open(".encryption_key").gets_to_end.to_s
+        else
+          nil
```

Comment:
An `encryption_key` shouldn't always exist, it does make sense though since I believe this will only get called if the encrypted file exists and the non encrypted file doesn't. Alternatively it could just load defaults if encrypted settings couldn't be decoded....